### DISCUSSION TITLE
How to get CLUSTER_CIDR and SERVICE_CIDR?

### DISCUSSION THREAD

**Discussion by ayr-ton** [submariner-io/submariner#1215] [Q&A] [answered]
Hey there, I'm using the helm chart installer, however I'm experiencing issues for getting the CLUSTER_CIDR and the SERVICE_CIDR.

There's some place/configmap/command where I can confirm these values for sure?

And the cluster id?

Thanks for your time replying this!

**Comment by sridhargaddam** [comment, marked as answer]
Hello Artyon 
 
> There's some place/configmap/command where I can confirm these values for sure?

Each CNI stores the cluster_cidr/service_cidr details in a different places. When using the operator based installation (using `subctl`), it internally tries to discover this info as shown here for various CNIs - https://github.com/submariner-io/submariner-operator/blob/f672e82a53b085430c71434f17e6db95814a2883/pkg/discovery/network/network.go#L64

> And the cluster id?

SUBMARINER_CLUSTERID is just a unique name that you give to your cluster and need not match with any existing info of your cluster.
